app/controllers/api/v1/spoonacular_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::SpoonacularController < ApiController
+
+  def search
+    plant_name = params["search_string"]
+
+    api_key = ENV["SPOONACULAR_API_KEY"]
+
+    url = "https://api.spoonacular.com/food/ingredients/search?apiKey=#{api_key}&query=#{plant_name}&number=1"
+
+    api_response = Faraday.get(url)
+
+    parsed_response = JSON.parse(api_response.body)
```

Comment:
nice work! I suggest refactoring by making a SpoonacularClient class. You can pull down the sample code from the Week 4 "Third Party API" exercise to guide you